samples/can/back_and_forth/arduino_can/arduino_can.ino

```diff
@@ -0,0 +1,78 @@
+// Copyright (c) Sandeep Mistry. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#include <CAN.h>
+
+bool readyToSend = true;
+
+void onReceive(int packetSize) {
+  // received a packet
+  Serial.print("Received ");
+
+  if (CAN.packetExtended()) {
+    Serial.print("extended ");
+  }
+
+  if (CAN.packetRtr()) {
+    // Remote transmission request, packet contains no data
+    Serial.print("RTR ");
+  }
+
+  Serial.print("packet with id 0x");
+  Serial.print(CAN.packetId(), HEX);
+
+  if (CAN.packetRtr()) {
+    Serial.print(" and requested length ");
+    Serial.println(CAN.packetDlc());
+  } else {
+    Serial.print(" and length ");
+    Serial.println(packetSize);
+
+    // only print packet data for non-RTR packets
+    while (CAN.available()) {
+      Serial.print(" ");
+      Serial.print(CAN.read(), HEX);
+    }
+    Serial.println();
+  }
+
+  Serial.println();
+
+  readyToSend = true;
+}
+
+void setup() {
+  Serial.begin(9600);
```

Comment:
In general, I like to see important hardcoded numbers at the top of a file in a constant for easy reference.